src/isolate_create_params.rs

```diff
@@ -24,6 +29,16 @@ pub struct CreateParams {
 }
 
 impl CreateParams {
+  /// Enables the host application to provide a mechanism for recording
+  /// statistics counters.
+  pub fn counter_lookup_callback(
```

Comment:
should be `set_counter_function` ?

https://source.chromium.org/chromium/chromium/src/+/main:v8/include/v8.h;l=9336-9340;drc=9aaf664253063e5a28fcacd58a67106357c3eaed

Reply:
That's the post-instantiation "set callback" method (which doesn't really work because you'll miss events.)

This is https://github.com/v8/v8/blob/a78cf618524ccf0cb5fb569f667f6e0bb6435a64/include/v8.h#L8337